stable/redis/README.md

```diff
@@ -356,8 +356,8 @@ By default, the chart mounts a [Persistent Volume](http://kubernetes.io/docs/use
 ### Existing PersistentVolumeClaim
 
 1. Create the PersistentVolume
-1. Create the PersistentVolumeClaim
-1. Install the chart
+2. Create the PersistentVolumeClaim
```

Comment:
**TIP:** If you always use `1.` for a numeric list, it is rendered properly. The advantage is that you don't need to rewrite all the indexes if you need to add a step in the middle.